docs/writing-stories/args.md

```diff
@@ -92,11 +94,11 @@ You can separate the arguments to a story to compose in other stories. Here’s
 
 <div class="aside">
 
-Note that if you are doing the above often, you may want to consider using [component-level args](#component-args).
+💡<strong>Note:</strong> If you find yourself applying this pattern often, you should consider using [component-level args](#component-args).
```

Comment:
As this is MDX, we could have an Aside component that accepts a `title` prop to handle this formatting automatically. Not a blocker for this effort, but something to keep in mind if we find ourselves doing this often.

Reply:
```suggestion
💡<strong>Note:</strong> If you find yourself re-using the same args for most of a component's stories, you should consider using [component-level args](#component-args).
```

Or something like that. Basically, "pattern" is pretty broad term. One could "apply this pattern often" in a way that doesn't fit the recommendation. This change attempts to make the language more specific to avoid that.